### PR TITLE
add KosyAbone

### PR DESCRIPTION
**Pull request by KosyAbone**
## Changes proposed

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1260"></a>

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hello, please revert the **`package-lock.json`** file, if you need any help feel free to ping us.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by KosyAbone** [comment]
> Hello, please revert the **`package-lock.json`** file, if you need any help feel free to ping us.
> 
> __ [sema-logo](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix




I have reverted the **`package-lock.json`** file. Kindly review.

**Comment by Pradumnasaraf** [comment]
> Hello, please revert the **`package-lock.json`** file, if you need any help feel free to ping us.
> 
> __ [sema-logo](https://semasoftware.com/gh)  

Hey! Kosy there are still modifications in the package-lock.json, can you please revert it?
Feel free to reach us out, for help__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by emmalearnscode** [comment]
If you check the diff you can see that it’s just the versions that are out of sync with each other. If you can update so that the versions match then this should be good to merge

**Comment by KosyAbone** [comment]
> If you check the diff you can see that it’s just the versions that are out of sync with each other. If you can update so that the versions match then this should be good to merge



Thank you Emma. I'll try and figure it out and fix.

**Comment by KosyAbone** [comment]
I believe it's all good now. Kindly review and merge. Thank you.

File: public/data/KosyAbone.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "Abone Kosiso",
+  "type": "personal",
+  "bio": "I am a fullStack Web Developer(MERN Stack), a coding instructor and an open source enthusiast. I love understanding how things work under the hood. I enjoy solving complex problems. Kindly connect with me to find out more about me...",
+  "avatar": "https://github.com/KosyAbone.png",
+  "links": [
+    {
+      "name": "Follow me on Github to see my work",
```
Comment (Panquesito7) [inline public/data/KosyAbone.json:8]: You might want to capitalize the `h` here. :)
```suggestion
      "name": "Follow me on GitHub to see my work",
```

Comment (KosyAbone) [reply public/data/KosyAbone.json]: Thank you. I have effected this change. 
All is good now. Kindly review

**Review by Panquesito7 — APPROVED** [review]

**Review by schmelto — APPROVED** [review]
lgtm

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by KosyAbone** [comment]
Thank you so much everyone for helping me out. I look forward to more contributions.

**Comment by KosyAbone** [comment]
I still cant see my profile show up on the live site http://linkfree.eddiehub.org/. Why could that be, please?

**Comment by emmalearnscode** [comment]
@KosyAbone it’s because the site doesn’t redeploy automatically when a profile gets added.

**Comment by KosyAbone** [comment]
> @KosyAbone it’s because the site doesn’t redeploy automatically when a profile gets added.

Oh I see. Thank you Emma😊

**Pull request merged**